src/kv/RocksDBStore.cc

```diff
@@ -756,32 +756,43 @@ int RocksDBStore::submit_common(rocksdb::WriteOptions& woptions, KeyValueDB::Tra
   return s.ok() ? 0 : -1;
 }
 
-int RocksDBStore::submit_transaction(KeyValueDB::Transaction t) 
-{
+void RocksDBStore::_submit_transaction(KeyValueDB::Transaction& t,
```

Comment:
You may return result in a regular manner via function return, no need for passing it by ref